draft-ietf-modpod-group-processes.md

```diff
@@ -345,15 +345,38 @@ NOT be shared between the teams.
 
 The Board of Directors of the IETF Administration LLC (IETF LLC) has
 fiduciary duty for the overall organization, which includes the duty
-to protect the organization from legal risk that may arise from
-illegal, vulgar, or manifestly harassing behavior of IETF participants.
-
-This protection MAY include the need for the LLC to take emergency moderation
-actions. These emergency actions are expected to be extremely rare, of temporary
-nature, and the incidents that required them SHOULD be immediately raised with
-the moderator team to let them determine any follow-up or more permanent
-moderation action. These incidents and the taken emergency action SHOULD also be
-communicated to the IETF community.
+to protect the organization from serious legal risk that may arise
+from the behavior of IETF participants.
+
+This protection MAY include the need for the IETF LLC to take
+emergency moderation actions. These emergency actions are expected to
+be taken only when the IETF LLC has received legal advice that such
+action is necessary, and therefore extremely rare in frequency. Some
+examples of where this might be necessary are:
+
+- Someone making credible threat of harm to other IETF participants.
+- Someone using IETF mailing lists and/or websites to share content
+  where publishing that content on IETF lists and/or websites brings
+  serious legal risk.
+- Someone making credible threats of legal action where any form of
+  interaction with them on IETF mailing lists may have serious legal
+  consequences for the IETF.
+
+If any such action is taken, the IETF LLC SHOULD inform the IESG as
+soon as possible and SHOULD provide full details of the subject of
+the action, nature of the action, reason for the action and expected
+duration, except where it receives legal advice to the contrary. The
+IETF LLC SHOULD also inform the moderator team and IETF community,
+except where it receives legal advice to the contrary.
```

Comment:
I would still avoid even SHOULDs.  Why not something like this?

```suggestion
If any such action is taken, the IETF LLC is requested to provide to the
IETF community full details of the subject(s), nature, reason, and expected duration
of the action in accordance with any legal advice it has received on the matter. 
```

Reply:
@elear it would be great if you could teach your editor not to gratuitously re-wrap lines. It makes the changes really hard to see.

Reply:
I agree with Jay. Whether it's SHOULD or should, it's more than a _request_ for transparency.

Reply:
I'm not going to stand on my head, but I would at least like the LLC Board to be comfortable with whatever language we're throwing at them.  Jay, can you ask?

Reply:
I don't think it's appropriate for me to involve the LLC board in this level of detail.  IETF Counsel are watching and will be asked for a formal review when it looks like this text has consensus for inclusion.

Reply:
So long as they don't need to own the results, I'm okay with this, but my entire approach is to be careful about placing demands on the LLC without the LLC being in a position to agree to those demands.  If you are in a position to agree on their behalf, fine.  Then we can close this comment.

Reply:
And again, I'm very happy to be in the rough on this.  It's truly the least of my concerns.

Reply:
Let's merge this. We can always modify later. @JayDaley are you still have pending edits?

Reply:
@larseggert Yes, one to move an instance of "except where it receives legal advice" to the front to cover the whole sentence.  Sending the commit now.

Reply:
@larseggert Ready to go.